### PR TITLE
Works section

### PR DESCRIPTION
**Pull request by MudasirSherwani**
**Works Section Done.**

- Sample Project Area 1,2,3 and 4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,20 @@
 > In the Mobile View Portfolio there are few important sections are visible.
 > This project has done after getting templete from FIGMA templete #4.
 
-![Portfolio in Mobile View](Protfolio_SS.PNG)
+![Portfolio in Mobile View](Protfolio_SS_1.PNG) (Protfolio_SS_12.PNG)
```
Comment (danmainah) [inline README.md]: - [ ] kindly make sure your image is displaying currently its not showing.

File: index.html [hunk 2]
```diff
@@ -61,14 +61,138 @@ <h4 class="heading">Skills</h4>
...
             </ul>
             </div>
+            <div class="nav-arrow">
+              <a href="#work"
+                ><img class="arrow" src="images/arrow.png" alt="" />
+              </a>
+              <a href="#work" class="works"><p>Work</p></a>
+            </div>
+      </section>
+      <section class="grid-container">
```
Comment (danmainah) [inline index.html:79]: - [x] Kindly check on your design, there is an empty space left on the right, also the background images are not well aligned kindly check the comparison below with the Figma design:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

- [x] Kindly ensure your buttons are clickable and follow the design below:

image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @MudasirSherwani , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by MudasirSherwani** [comment]
Hi Daniel Maina,
I'm facing a problem in resolving the issue you find out,
The project is displaying fine at Mozilla but the problem you identified
from Chrome.
please suggest to me a solution that how can I fix it.
Thank You!
Regards: Mudasir Sherwani

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @MudasirSherwani 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**